pipeline/terraform/stack/service_work_router.tf

```diff
@@ -14,9 +14,8 @@ module "router" {
   container_image = local.router_image
 
   security_group_ids = [
-    # TODO: Do we need these?
+    # TODO: Do we need the egress security group?
```

Comment:
Is the answer not "yes, we need it for the logging cluster access"?

Reply:
Not any more, because we talk to the logging cluster over a VPC endpoint.